realm/realm-library/src/main/java/io/realm/internal/sync/OsSubscription.java

```diff
@@ -24,6 +24,7 @@
 import io.realm.internal.ObserverPairList;
 import io.realm.internal.OsResults;
 
+@KeepMember
```

Comment:
Use `@Keep` the class (like for `SyncManager` & `SyncSession`)  `@KeepMember` is for methods

Reply:
Yes, but `@Keep` keeps all class members, we are only interested in one method.
`@KeepMethod` is also required on the class to make the Proguard rules work. They are:
```
-keep class io.realm.internal.KeepMember
-keep @io.realm.internal.KeepMember class * { @io.realm.internal.KeepMember *; }
```

Where as this is the rules for `@Keep`:
```
-keep class io.realm.internal.Keep
-keep @io.realm.internal.Keep class * { *; }
```